dali/dfu/dfurun.cpp

```diff
@@ -1604,7 +1604,12 @@ class CDFUengine: public CInterface, implements IDFUengine
                         if (needrep)
                             feedback.repmode=cProgressReporter::REPbefore;
                         fsys.import(fdesc, dstFile, recovery, recoveryconn, filter, opttree, &feedback, &abortnotify, dfuwuid);
-                        if (!abortnotify.abortRequested()) {
+
+                        // Check replication if and only if import was not aborted and it was complete and
+                        // any file match to wildcard
+                        if (!abortnotify.abortRequested() &&
+                            recovery->getPropBool("@complete", false ) && !recovery->getPropBool("@noFileMatch", false ))
```

Comment:
extra spaces between 'false' and ')'

Reply:
Is this condition correct?

What happened in 6.2 when you sprayed a wildcard with 0 files ? 
I had thought it created a logical file that had 0 records.. ?

Reply:
I tested the 6.2 and yes it generated an empty logical file, its (empty) physical parts and replication of the physical parts.
Now my modified code behave in same way: from an empty directory it generates same result.